Oware/src/Move.java

```diff
@@ -1,12 +1,27 @@
 
 public class Move
 {
-	// return results from
-
-	public static void main(String[] args)
+	private Board currentBoard;
+	private int currentPosition;
+	
+	public void makeMoveUsingPitNum(Board board, int pitNum)
 	{
-		// TODO Auto-generated method stub
-
+		currentBoard = board;
+		currentPosition = pitNum;
+		int movableSeeds = currentBoard.getSeedTotalfromPit(pitNum);
+		currentBoard.emptyPit(pitNum);
+		for(int i=0; i<movableSeeds; i++)
+		{
+			if(currentBoard.isPitExcist(pitNum) == true)
+			{
+				currentBoard.putSeedInPit(1, pitNum);
```

Comment:
So, it looks like we are going to put all the seeds in the same pit,  where do we put the next seed into the next pit?